.github/workflows/gui.yml

```diff
@@ -512,6 +512,7 @@ jobs:
           ENSO_TEST_USER: ${{ secrets.ENSO_CLOUD_TEST_ACCOUNT_USERNAME }}
```

Comment:
Ignoring results of the tests is the right fix for now. To quote [my note in another PR](https://github.com/enso-org/enso/pull/11594/commits/383e94195ce024609fdc7835325f30c9bd57d5c0#r1849946447):

This file is labeled with

```yaml
 # This file is auto-generated. Do not edit it manually!
# Edit the enso_build::ci_gen module instead and run `cargo run --package enso-build-ci-gen`.
```

that's why I believe it is a **good idea** to remove this section manually. Next time someone tries to run `cargo run --package enso-build-ci-gen` this section will be regenerated back.

A subsequent PR should fix the tests and execute `cargo run --package enso-build-ci-gen` to enable them again.

Reply:
I don’t believe changing auto-generated files and introducing a hidden discrepancy between the code and workflow is justified here. To make an example:
- if we have _two_ different CI issues, and we fix both of them as you are suggesting
- I fix one of them and rerun the generator as suggested
- I suddenly also disabled mitigation of the second issue I’m not even aware of